pages/events/index.vue

```diff
@@ -31,19 +31,32 @@ import QiskitPage from '~/components/qiskit/QiskitPage.vue'
 
 export default class extends QiskitPage {
   routeName: string = 'events'
+  windowWidth: Number = 0
 
   autoplayVideo () {
+    if (!this.$refs.video) {
+      return
+    }
+
+    const video = this.$refs.video as any
+
     setTimeout(() => {
-      if (this.$refs.video) {
-        this.$refs.video.load()
-        this.$refs.video.muted = true
-        this.$refs.video.play()
-      }
+      video.load()
+      video.muted = true
+      video.play()
     }, 1)
```

Comment:
You don't need to make this process "async" since it is already being used in an async way in the `mounted()` function when called after `nextTick()`.

Reply:
Done 👍  Thanks!